src/v2/ion/i18nTransformer.js

```diff
@@ -155,6 +155,7 @@ const I18N_OVERRIDE_MAPPINGS = {
   'api.users.auth.error.POST_PASSWORD_UPDATE_AUTH_FAILURE': 'oie.post.password.update.auth.failure.error',
   'security.access_denied': 'errors.E0000006',
   'api.factors.error.sms.invalid_phone': 'oie.phone.invalid',
+  'app.ldap.password.reset.failed': 'errors.E0000017'
```

Comment:
Do we know if this key is sent back on for OIE or is it for V1 as well ? Asking because we try to differentiate by adding the `oie.` prefix for OIE only keys. We can leave it as is, if it is used in V1 as well.

Reply:
@nikhilvenkatraman-okta according to this line: https://github.com/okta/okta-core/blob/master/components/apps/directories/impl/src/main/java/com/saasure/application/ldap_sun_one/auth/SunONELoginHandlerImpl.java#L1263, it looks like this is used in V1 as well.